features/FxA-97-password-strength/README.md

```diff
@@ -0,0 +1,67 @@
+
+## Hypothesis
+
+[comment]: # (A high level hypothesis of how the feature you're proposing is going help us achieve the outcomes listed above. I recommend this be a sentence of the form. We believe that doing this building this creating this experience for these people will achieve this outcome. We will know this is true when we see this qualitative feedback quantitative feedback KPI change.)
+
+We believe that providing more information about password strength for registering users will achieve stronger passwords. We will know this is true when we see a reduction in our bloom filter numbers.
+
+We believe that providing more information about password strength for registering users will achieve greater awareness about password strength. We will know this is true when we survey new users and ask about passwords.
+
+We believe that blocking known weak passwords will achieve stronger passwords. We will know this is true when we see a reduction in our bloom filter numbers.
+
+We believe that these changes will result in us collecting only the data that benefits users. We will know this is true when we see a slight reduction in the number of new accounts.
+
+## Metrics
+
+[comment]: # (How are you going to measure the outcome / success? Please provide sample artifact graphs here.)
+
+* Datadog Dashboard › FxA Content Server - Password Strength › % of users that hit each restriction
+ * Fewer users hitting ALL_LETTERS_OR_NUMBERS and BLOOMFILTER_HIT
+* Datadog Dashboard › FxA Content Server - Registration › % of too short passwords
+ * Fewer users hitting TOO_SHORT
+* Datadog Dashboard › FxA Content Server - Registration › Signup funnel
+ * Slightly reduced signup rate
+
+## Detailed design
+
+[comment]: # (This is the bulk of the RFC. Explain the design in enough detail for somebody familiar with the language to understand. This should get into specifics and corner-cases, and include examples of how the feature is used.)
+
+![Password Strength](password-strength.png)
+
+#### Password field give focus
+When the user gives focus to the password field on the registration form, show instructions below that say:
+* A strong, unique password will keep your Firefox data safe from intruders. <u>More info</u>.
+
+"More info" will link to a modal dialog containing concise instructions on creating a strong password based on [this SUMO article](https://support.mozilla.org/en-US/kb/create-secure-passwords-keep-your-identity-safe)
+
+#### User enters characters
+When the user enters any characters, the instructions change to become:
+* 8 characters minimum, but longer if you plan to sync passwords. <u>More info</u>.
```

Comment:
`but longer if you plan to sync passwords` sounds like we are going to enforce the `longer if you plan to sync passwords` part. Perhaps something like `8 characters minimum, but we recommend longer if you plan to sync passwords.`